README.md

```diff
@@ -1,3 +1,532 @@
 _Note that this project depends on internal Symphony infrastructure (repository.symphony.com), and therefore it can only be built by Symphony LLC employees/partners._
 
-# Github Integration
+# GitHub WebHook Integration
+The GitHub Webhook Integration will allow you to receive notifications in Symphony whenever you make a code-push, pull request, comments on pull requests, issues, comments on issues, merges, and the latest deployment status.
+
+## [Build Instructions](#build-instructions-for-the-java-developer)
+
+## How it works
+As a GitHub admin of a repository you can configure a WebHook to post messages in a URL you generate in the GitHub WebHook Application available on Symphony Market and then you'll start receiving notifications for the supported events.
+As of the current version, one must configure the WebHook **Content Type** as *application/json* as shown below:
+
+![Selecting content type](src/docs/sample/sample_webhook_content_type.png)
+
+## What formats and events it supports and what it produces
```

Comment:
Add information about the dependency the IB has on the github API to retrieve user information, and indicate that if the integration bridge host does not have access to the server, the github users may not have username information on the event displayed on the chat. This might be the case when the github server is deployed on prem and for some reason is not accessible from the IB host.